packages/ffe-form-react/src/PhoneNumberField.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {string, bool, func} from 'prop-types';
+
+import i18n from './i18n/i18n';
+
+export default function PhoneNumberField({
+                               phone,
```

Comment:
Again more a style preference, but when destructuring this much I prefer doing it outside the function signature:

```js
export default function PhoneNumberField(props) {
    const {
        phone,
        countryCode,
        onChange,
        onBlur,
        locale,
        disabled,
        showValidationError,
        className,
    } = props;
```